### PR TITLE
Add Exists filter. 

### PR DESCRIPTION
**Pull request by dereuromark**
Also fixed bad inflection and wrong filter result return value.

We often have to filter by content of rows (filled or not yet filled), and have a dropdown with
```php
// For a specific row field
'' => ' - show all - '
'1' => ' content set '
'0' => ' empty '
```
This Exists one takes the heavy load of a manual custom filter away as this easily does this based on the correct empty comparison for the ORM.
One improvement could be auto-detect of nullValue (schema not null => empty string), but maybe thats a bit too much overhead then.
So I settled for this compromise.

A new minor is OK?

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -15,5 +15,4 @@ charset = utf-8
 end_of_line = crlf
 
 [*.yml]
-indent_style = space
```
Comment (ADmad) [inline .editorconfig]: Why this change? YAML files are space indented.

File: src/Model/Filter/FilterCollection.php [hunk 2]
```diff
@@ -74,7 +74,7 @@ public function add($name, $filter, array $options = [])
     protected function _loadFilter($name, $filter, array $options = [])
     {
         if (empty($options['className'])) {
-            $class = Inflector::classify($filter);
```
Comment (ADmad) [inline src/Model/Filter/FilterCollection.php]: Why this change?

**Comment by ADmad** [comment]
> Also fixed bad inflection and wrong filter result return value.

Please make a separate PR for these so that a new bugfix release can be made. Then we can make a new minor with the new filter.

File: .editorconfig (hunk 1 above)
Comment (dereuromark) [reply .editorconfig]: Your ide will tell u: not necessary here.

File: src/Model/Filter/FilterCollection.php (hunk 2 above)
Comment (dereuromark) [reply src/Model/Filter/FilterCollection.php]: Revert and u see: bad code killing the name resolvement for things that are plural or appear as such

**Comment by dereuromark** [comment]
IMO those can and should be shipped together.

**Comment by ADmad** [comment]
The bug fixes and new filter are unrelated. The bugfixes should first go in a bugfix release. After that a new minor with `Exists` filter.

File: src/Model/Filter/FilterCollection.php (hunk 2 above)
Comment (ADmad) [reply src/Model/Filter/FilterCollection.php]: That's what tests are for :wink:.

**Comment by dereuromark** [comment]
I wanted keep the process lean here. A bugfix can always go into a minor just fine.
Feel free to extract them, but people could even have relied on certain wrong behavior .
Shipping this in this minor then even makes more sense than patching it out.

**Comment by ADmad** [comment]
You won't have to come up with justifications if you simply stop clubbing unrelated changes in same PR.

File: .editorconfig (hunk 1 above)
Comment (ADmad) [reply .editorconfig]: The `editorconfig` file is meant to tell the formatting style to IDE.

Comment (ADmad) [reply .editorconfig]: The proper justification for removing this is it's redundant since we already have `indent_style = space` under `[*]`.
